cmd/nri-kubernetes/main.go

```diff
@@ -28,6 +20,12 @@ import (
 	kubeletClient "github.com/newrelic/nri-kubernetes/v2/src/kubelet/client"
 	"github.com/newrelic/nri-kubernetes/v2/src/prometheus"
 	"github.com/newrelic/nri-kubernetes/v2/src/sink"
+	"github.com/sethgrid/pester"
```

Comment:
NIT: order of imports

Reply:
This is actually formated with go imports see that is sorted correctly

Reply:
fixed